src/SFA.DAS.ApprenticeCommitments.Web/Pages/DateModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Pages
+{
+    /// <summary>
+    ///     Encapsulates a date whilst exposing separate day, month and year properties that can
+    ///     be set independently.  The three elements are validated when set but may not be valid
+    ///     when used together, for example 31-Feb. In this case accessing the <see cref="Date"/>
+    ///     will throw an exception. To avoid this check the <see cref="IsValid"/> property before
+    ///     accessing <see cref="Date"/>.
+    /// </summary>
+    public class DateModel
+    {
+        public DateModel()
+        {
+        }
+
+        public DateModel(DateTime date) : this(date.Year, date.Month, date.Day)
+        {
+        }
+
+        public DateModel(int year, int month, int day) =>
+            (Year, Month, Day) = (year, month, day);
+
+        public virtual int Day { get; set; }
+        public virtual int Month { get; set; }
+        public virtual int Year { get; set; }
+
+        public DateTime Date =>
+            IsValid ? new DateTime(Year, Month, Day) : default;
+
```

Comment:
IIRC we discussed this on Friday - the rationale for using default here is that we want to defer validation to the API.  My memory of why we did this may be fuzzy though.  Looking at it now, maybe a nullable `DateTime` make this clearer. 🤔

Reply:
no leave it as is for teh moment